frontend/server/src/DAO/QualityNominations.php

```diff
@@ -190,6 +190,76 @@ private static function getVotesForNomination(int $qualitynomination_id) {
         return $votes;
     }
 
+    /**
+     * Returns the quality seal and id of a nomination for a given problem and user.
+     *
+     * @return array{quality_seal: bool, qualitynomination_id: int}
+     */
+    public static function getReviewedData(
+        int $problemId,
+        int $userId
+    ): array {
+        $sql = "
+            SELECT
+                qn.contents,
+                qn.qualitynomination_id
+            FROM
+                QualityNominations qn
+            INNER JOIN
+                Identities i ON i.user_id = qn.user_id
+            WHERE
+                nomination = 'quality_tag' AND
+                i.user_id = ? AND
+                qn.problem_id = ?;";
+        $params = [
+            $userId,
+            $problemId,
+        ];
+
+        /** @var array{contents: string, qualitynomination_id: int}|null */
+        $row = \OmegaUp\MySQLConnection::getInstance()->GetRow($sql, $params);
+
+        if (is_null($row)) {
+            return [
+                'quality_seal' => false,
+                'qualitynomination_id' => 0,
```

Comment:
```suggestion
                'qualitynomination_id' => null,
```

hay que intentar evitar valores mágicos. si un dato no existe, hay que indicar eso explícitamente con `null` en vez de números.